### PR TITLE
Update external-email-forwarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information on Automatic as per Microsoft recent changes on External email forwarding controls and policy change.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3147

File: microsoft-365/security/office-365-security/external-email-forwarding.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ description: "."
 
 External forwarding is controlled by the *outbound anti-spam policy* and scoped to users based on the configured setting. Currently 3 settings are supported:
 
-- **Automatic** – This is system-controlled: It allows outbound spam filtering to control automatic external email forwarding. This is the default setting.
+- **Automatic** – “Automatic” setting will block automatic external forwarding. Internal automatic forwarding of messages will continue to work. This is the default setting.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/external-email-forwarding.md]:
```suggestion
- **Automatic** – Automatic external forwarding is blocked. Internal automatic forwarding of messages will continue to work. This is the default setting.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**